views/js/filterData.js

```diff
@@ -0,0 +1,79 @@
+function genreateTextFilterHTML(question) {
+    return `
+        <div class="containFilter">
+            <div class="filterName">
+                <span class="question">${question.question}</span>
+                <span>${question.isNumber ? ' (Higher will show)' : ''}</span>
+                <i class="fas fa-caret-down"></i>
+            </div>
+            <input isNumber=${question.isNumber} onKeyUp="applyFilters()" whatType=${question.type} whatName=${question.name} type="text" class="filterInput" placeholder="Search" class=" class="searchForText" />
+        </div>
+    `
+}
+
+function generateFiltersHTML(data, selector) {
+    var htmlArry = [];
+    for (var question of data) {
+
+        if (question.type == 'text' || question.type == 'textarea') {
+            htmlArry.push(genreateTextFilterHTML(question));
+            continue;
+        }
+        /*
+        htmlArry.push(
+          `<div>${question.name}</div>`
+       );
+       */
+    }
+    document.querySelector(selector).innerHTML = htmlArry.join('');
+}
+
+function getAllFilterValues() {
+    var filterValues = {};
+    var inputs = document.querySelectorAll('.filterInput');
+    for (var input of inputs) {
+        if (input.value == '') continue;
+        filterValues[input.getAttribute('whatName')] = {'value': input.value, 'type': input.getAttribute('whatType')};
+        if (input.getAttribute('isNumber') == 'true') filterValues[input.getAttribute('whatName')].isNumber = true;
+    }
+    return filterValues;
+}
+
+function generateUsersHTML(users) {
+    var htmLArry = [];
```

Comment:
// use "/image/${user.form.formId}" to get the image but now use base 64 image/url image